src/lwc/edaSetupHealthCheckRun/edaSetupHealthCheckRun.html

```diff
@@ -0,0 +1,37 @@
+<template>
+    <lightning-card title={labelReference.stgEDASettingsHealthCheckTitle}>
+        <div class="slds-grid slds-wrap">
+            <div class="slds-col 
+                        slds-size--1-of-1 
+                        slds-p-horizontal_small 
+                        slds-p-vertical_small
+                        slds-text-body_regular">
+                <p>{labelReference.stgHealthCheckDescription}</p>
+            </div>
+
+            <div class="slds-col 
+                        slds-size--1-of-1 
+                        slds-medium-size--1-of-2 
+                        slds-large-size--1-of-2 
+                        slds-text-align_left
+                        slds-text-body_small 
+                        slds-p-horizontal_small
+                        slds-p-vertical_small">
+                {lastRunDate}
+            </div>
+            <div class="slds-col 
```

Comment:
To right align these, consider using `slds-col_bump-left slds-text-align_right`

Reply:
The spec indicates that the button should be aligned left, is this a typo?

Reply:
That is accurate